src/core/js/models/adaptModel.js

```diff
@@ -467,7 +467,10 @@ define([
                 try {
                     var model = Adapt.findById(id);
 
-                    if (!model.get("_isAvailable")) continue;
+                    this.stopListening(model, "change:_isComplete");
```

Comment:
it also needs to be changed to include the callback function.
this call to stopListening will remove all "change:_isComplete" event listeners attached to this model, which can override subsequently added behaviour.
```
this.stopListening(model, "change:_isComplete", this.onIsComplete);
```